oa2osm.js

```diff
@@ -59,28 +59,29 @@ function toTitleCase(str) {
 
 let count = 0;
 
-const parser = parse({ columns: true });
 const transformer = transform((record, callback) => {
+  const parsedRecord = JSON.parse(record);
   count++;
   if (count % 100 == 0) {
     process.stdout.write(" " + numeral(count).format('0a') + "   \r");
   }
   const e = [{
     _attr: {
-      lat: record.LAT,
-      lon: record.LON,
+      lat: parsedRecord.geometry.coordinates[0],
+      lon: parsedRecord.geometry.coordinates[1],
```

Comment:
```suggestion
      lat: parsedRecord.geometry.coordinates[1],
      lon: parsedRecord.geometry.coordinates[0],
```

GeoJSON is lon, lat order

Reply:
Fixed, thanks I didn't realise this